### PR TITLE
states and add data functions

### PR DESCRIPTION
**Pull request by iamvldmrbrvkv**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useState } from "react";
```
Comment (nothingisfunny) [inline src/App.js:1]: Very good 👍

File: src/App.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React from "react";
+import React, { useState } from "react";
 import { RouterProvider, createBrowserRouter, createRoutesFromElements, Route, Navigate } from "react-router-dom"
```
Comment (nothingisfunny) [inline src/App.js:2]: test

**Comment by nothingisfunny** [comment]
🚢

**Comment by iamvldmrbrvkv** [comment]
> 🚢

❓

**Comment by iamvldmrbrvkv** [comment]
> 🚢

👨‍✈️

**Pull request merged**